astro/src/content/blog/llms-for-docs.mdx

```diff
@@ -0,0 +1,231 @@
+
+We don't just have our techdocs ingested in Kapa, we also ingest our [OpenAPI spec](https://github.com/FusionAuth/fusionauth-openapi), our [YouTube channel](https://www.youtube.com/c/fusionauth), our [forum posts](/community/forum/) and a number of other sources. Kapa has answered tens of thousands of queries, both for external and internal users. It's great for discovery of documentation or other resources.
+
+Make sure whatever chatbot you use can say "I don't know" and includes links to sources. We've had few reported hallucinations, but they do happen. Having the source means users can verify the answers.
+
+### Implementation Details
+
+Find an LLM chatbot vendor and follow their installation instructions.
+
+### Success Metrics
+
+The measure of success here is twofold:
+
+* are people using this tool
+* are they finding the answers they need
+
+Both of these metrics should be delivered by any chatbot you use or. If you do end up building a solution (don't do it!) make sure you plan to build this reporting.
+
+## Implement llms.txt: The Emerging Standard for AI Discoverability
+
+Another way you can help developers using LLMs succeed with your docs is to build an `llms.txt` file.
+
+[`llms.txt` is a relatively new standard](https://llms-txt.org/) analogous to the `sitemap.xml` or `robots.txt` files, but for LLMs rather than search engine crawlers. It has a different format to provide maximum content in minimum number of tokens. Markdown is used rather than HTML.
+
+There are actually two files:
+
+* `llms.txt` is a list of URLs with useful content, organized by section. Here's our [docs `llms.txt` file](/docs/llms.txt). 
+* `llms-full.txt` is a similar list but includes the full text rather than just the URL of the pages. Here is our [docs `llms-full.txt` file](/docs/llms-full.txt). 
+
+It's unclear if you can have more than one set of `llms.txt` files for different sections of your site; the standard is still evolving. [Here's an open issue to discuss this](https://github.com/AnswerDotAI/llms-txt/issues/59).
+
+These files are typically located at the root of your domain: `https://example.com/llms.txt` for example. This is so LLMs have one place to look.
+
+If you put these files somewhere else or have more than one of them, add a link from your HTML pages to the files. Crawlers for LLMs hunt around, but not too much. We shipped an `llms.txt` file for our docs in December, but it wasn't at the root location above. This file was downloaded by the OpenAI chatbot nearly 60 times as much after we linked to it from our HTML docs pages as before we did.
+
+In addition, when you make these files available to developers browsing your site, they can retrieve it and upload it to an LLM of their choosing. This allows them to interrogate the docs or learn more about the doc structure using their tool of choice.
```

Comment:
`these files` are being referred to as "it" in the rest of the sentence. Change "it" to "them" to maintain consistent plurality.